crates/sui-benchmark/tests/simtest.rs

```diff
@@ -512,15 +517,15 @@ mod test {
                     );
                 },
                 PerObjectCongestionControlMode::TotalGasBudgetWithCap => {
-                    let total_gas_limit = checkpoint_budget_factor
-                        * DEFAULT_VALIDATOR_GAS_PRICE
-                        * TEST_ONLY_GAS_UNIT_FOR_HEAVY_COMPUTATION_STORAGE;
                     config.set_max_accumulated_txn_cost_per_object_in_narwhal_commit_for_testing(total_gas_limit);
                     config.set_max_accumulated_txn_cost_per_object_in_mysticeti_commit_for_testing(total_gas_limit);
                     config.set_gas_budget_based_txn_cost_cap_factor_for_testing(total_gas_limit/cap_factor_denominator);
                 },
             }
             config.set_max_deferral_rounds_for_congestion_control_for_testing(max_deferral_rounds);
+            config.set_max_txn_cost_overage_per_object_in_commit_for_testing(
+                allow_overage_factor * total_gas_limit,
+            );
```

Comment:
Given the randomization I felt this was probably good enough, "overage" mode doesn't really make much sense for TotalTxCount mode anyway since you could only ever go "over" by exactly one